tiledb/sm/filesystem/s3.cc

```diff
@@ -181,6 +181,9 @@ Status S3::init(
                                                   Aws::Http::Scheme::HTTPS;
   config.connectTimeoutMs = s3_config.connect_timeout_ms_;
   config.requestTimeoutMs = s3_config.request_timeout_ms_;
+  config.caFile = s3_config.ca_file_;
```

Comment:
Do these need to be `.c_str()` or alternatively the AWS string typedef? I recall some edge case a while ago with differing string types -- maybe @joe-maley remembers this.

Reply:
Works locally and on CI, but now I see that all of the other assignments use `c_str` (or construct an `Aws::String` directly). Will change to that just to be safe -- maybe compiler-specific?